src/utilities/CatalogApiProcessed.ts

```diff
@@ -2,16 +2,37 @@ import * as AST from "ast_wrapper";
 import {CARTA} from "carta-protobuf";
 import {ProcessedColumnData} from "utilities";
 import {AppStore, NumberFormatType, SystemType} from "stores";
+import {CatalogSystemType} from "models";
 
-enum SimbadType {
+enum DataType {
     CHAR = "CHAR",
     SHORT = "SHORT",
     FLOAT = "FLOAT",
     LONG = "LONG",
     INT = "INT",
-    DOUBLE = "DOUBLE"
+    DOUBLE = "DOUBLE",
+    UNSIGNEDBYTE = "UNSIGNEDBYTE"
 }
 
+export type VizieResource = {
```

Comment:
typo? `VizieRResource`?

I think perhaps we should be consistent and use a lowercase `r` in `vizier` everywhere except user-facing strings.